huggle/resources.cpp

```diff
@@ -58,6 +61,26 @@ QByteArray Huggle::Resources::GetResourceAsBinary(QString path)
     return result;
 }
 
+QString Huggle::Resources::GetRandomProTip()
+{
+    if (protips.isEmpty())
+    {
+        QString tips = GetResource("/huggle/resources/Resources/huggle_tips.txt");
+        protips = tips.split("\n", ::QString::SkipEmptyParts);
+    }
+
+    if (protips.isEmpty() || protips.count() == 1)
+        return "Inform whoever packaged this Huggle that they did it wrong. There is no tips db.";
+
+    int random = GetRandom(0, protips.count() - 1);
+    while (last_tip != -1 && last_tip == random)
```

Comment:
I would think the last_tip != -1 check is redundant? If it is false (i.e., last_tip is equal to -1) then the other clause (last_tip == random) should also be false